clifford/test/test_complex.py

```diff
@@ -0,0 +1,86 @@
+
+import pytest
+import numpy as np
+import clifford as cf
+
+from clifford import Cl, conformalize
+
+
+# using fixtures here results in them only being created if needed
+@pytest.fixture(scope='module')
+def g2():
+    return Cl(2)[0]
+
+
+@pytest.fixture(scope='module')
+def g3():
+    return Cl(3)[0]
+
+
+@pytest.fixture(scope='module')
+def g4():
+    return Cl(4)[0]
+
+
+@pytest.fixture(scope='module')
+def g5():
+    return Cl(5)[0]
+
+
+@pytest.fixture(scope='module')
+def g4_1():
+    return Cl(4, 1)[0]
+
+
+@pytest.fixture(scope='module')
+def g3c():
+    return conformalize(Cl(3)[0])[0]
```

Comment:
Using both seems unnecessary, given you're not actually using any of the other methods.